source/frontend/src/features/mapSideBar/lease/tabs/consultations/detail/ConsultationListView.tsx

```diff
@@ -151,6 +151,21 @@ export const ConsultationListView: React.FunctionComponent<IConsultationListView
                     </div>
                   }
                 >
+                  {consultation?.consultationTypeCode?.id === 'OTHER' && (
```

Comment:
**nit:** would prefer to see OTHER be modelled as an `API_CodeGen_Blah.OTHER` as we have been doing recently with auto-generated values but this is also fine.